packages/guui/README.md

```diff
@@ -0,0 +1,187 @@
+# Guui
+
+A set of guardian-styled components and helpers
+
+**Alpha package - under construction 🚧**
+
+This package is still under heavy construction, please do not install this in applications intended for
+use in production. The API may change significantly before it is out of alpha. If you enjoy working on shifting sands and wish to use this package in your application, please talk to the Platform team, or email dotcom.platform.
+
+## Install
+
+```bash
+$ npm install @guardian/guui
+```
+
+```bash
+$ yarn add @guardian/guui
+```
+
+## Usage
+
+GUUI components are [REACT](https://www.google.com) components that use [EMOTION](https://emotion.sh/docs/introduction) for writing their associated styles in JavaScript (css-in-js).
+
+`guui` contains a library of reusable components that you can import into your own `React` application. It also contains a number of helpers for creating your own `guui` style components and rendering them on the client and the server.
+
+### Prerequisites
+
+When importing or creating a component with `guui` you must ensure `React` is available in your module. Why? because the JSX a component such as:
+
+```js
+<Headline/>
+```
+
+Will be transpiled to:
+
+```js
+React.createElement(Headline)
+```
+
+**How to can you ensure `React` is available?**
+
+**Option 1:** Import your own `React` dependency.
+
+```js
+import React from 'react'
+```
+
+**Option 2:** Import the version of `React` exported by `guui`.
+
+```js
+import { React } from '@guardian/guui';
+```
+
+**Option 3:** Automatically load `React` instead of having to import or require it everywhere.
+
+```js
+plugins: [
+    new webpack.ProvidePlugin({
+        React: 'react'
+    })
+]
+```
+
+#### Importing a component from `guui`
+
+```js
+import { React } from '@guardian/guui';
+import CloseButton from '@guardian/guui/components/Close';
+
+export default function MyComponent() {
+    return (
+        <CloseButton
+            foregroundcolor={'black'}
+            backgroundcolor={'yellow'}
```

Comment:
You don't need `{'curly braces'}` around strings

Reply:
hah yeah, cut and paste that code from elsewhere and swapped out the variables previously inside the braces for strings.